service_container/service_subscribers_locators.rst

```diff
@@ -516,6 +516,22 @@ of the ``key`` tag attribute (as defined in the ``index_by`` locator option):
 
 .. configuration-block::
 
+    .. code-block:: php-attributes
+
+        // src/CommandBus.php
+        namespace App;
+
+        use Symfony\Component\DependencyInjection\Attribute\TaggedLocator;
+        use Symfony\Component\DependencyInjection\ServiceLocator;
+
+        class CommandBus
+        {
+            public function __construct(
+                #[TaggedLocator('app.handler', indexAttribute: 'key')] ServiceLocator $locator
```

Comment:
```suggestion
                #[TaggedLocator('app.handler', indexAttribute: 'key')]
                ServiceLocator $locator
```

Lets go with a line break here and in the other examples

Reply:
Agree! Here you go 👍